ntbs-integration-tests/TransferPages/ActionTransferPageTests.cs

```diff
@@ -12,6 +16,29 @@ public class ActionTransferPageTests : TestRunnerNotificationBase
         protected override string NotificationSubPath => NotificationSubPaths.ActionTransferRequest;
         public ActionTransferPageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory) { }
 
+        public static IList<Notification> GetSeedingNotifications()
+        {
+            return new List<Notification>()
+            {
+                new Notification
+                {
+                    NotificationId = Utilities.NOTIFIED_ID_WITH_TRANSFER_REQUEST_TO_REJECT,
+                    NotificationStatus = NotificationStatus.Notified,
+                    // Requires a notification site to pass full validation
+                    NotificationSites = new List<NotificationSite>
+                    {
+                        new NotificationSite { NotificationId = Utilities.NOTIFIED_ID_WITH_TRANSFER_REQUEST_TO_REJECT, SiteId = (int)SiteId.PULMONARY }
+                    },
+                    Episode = new Episode
+                    {
+                        TBServiceCode = Utilities.TBSERVICE_ABINGDON_COMMUNITY_HOSPITAL_ID,
+                        HospitalId = Guid.Parse(Utilities.HOSPITAL_ABINGDON_COMMUNITY_HOSPITAL_ID),
+                        CaseManagerUsername = Utilities.CASEMANAGER_ABINGDON_EMAIL
+                    }

```

Comment:
Do you need these additional parts for your testing?

Reply:
Yes :)